reviewrot/base.py

```diff
@@ -0,0 +1,68 @@
+import datetime
+from dateutil.relativedelta import relativedelta
+
+class BaseService(object):
+    
+    def __init__(self):
+        pass
+    
+    def find_duration(self, rel_diff):
```

Comment:
You should add a docstring to this method. It's not obvious what it does from the name or the code. In fact, I suggest a name change. How about ```format_duration``` instead?